packages/react/src/components/Authenticator/Router/index.tsx

```diff
@@ -46,7 +48,11 @@ export function Router({
 
   // `Authenticator` might not have `children` for non SPA use cases.
   if (['authenticated', 'signOut'].includes(route)) {
-    return children ? children({ signOut, user }) : null;
+    return children
+      ? typeof children === 'function'
```

Comment:
Fairly certain that both potential `typeof` values for `children` are `function`?

Reply:
When I was trying this out with `console.log(children, typeof children)`, 

I got `typeof children === 'object'` for 

```jsx
<Authenticator >
  <div>asdf</div>
</Authenticator>
```

and `typeof children === 'function'` for

```jsx
<Authenticator >
  {({ signOut }) => <button onClick={signOut}>Sign out</button>}
</Authenticator>